src/azure/basic-azure-cli-facade.ts

```diff
@@ -186,6 +203,20 @@ export class BasicAzureCliFacade implements AzureCliFacade {
           "Subscription cost could not be requested via the Cost Management API",
         );
       }
+
+      errMatch = this.errNotAuthorized.exec(result.stderr);
+      if (errMatch) {
+        throw new MeshAzurePlatformError(
```

Comment:
Ah I see you do this here. I would just not throw an error at all but rather make a warning. (Okay it would be weird though when this happens during listing tenants...)

Reply:
but we need to throw so I can return an empty Promise in the method. I know Johannes is not in favor to do "flow control" with exceptions with could be arguably the case here. But on the other hand its a sound mechanism for handling exceptional errors. And I think missing permissions could be considered an exception here. I am just a bit lazy (and I think its overkill to add another signaling path with like `checkPermission(response): bool` or something just to handle it...